### ISSUE TITLE
Bug in regeneration of code in typed_backend

### ISSUE THREAD

**Issue by rickteuthof** [leegbestand/coconut#8]
When generating code once, it works fine, however, once the code is regenerated, it throws a segmentation fault. This happens somewhere around line 261 in driver.c.

**Comment by leegbestand** [comment]
Yes, you need to disable the hashing. Just disabling that if-block around 254 works.

**Comment by leegbestand** [comment]
But the typed backend is a WIP, because it will containt the traversal rewrite and all. If you want to compare, you can use the old repo.
